ggircs-app/app/components/FuelManagement/MappedFuelTypeTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { Table } from "react-bootstrap";
+import Button from "@button-inc/bcgov-theme/Button";
+import Alert from "@button-inc/bcgov-theme/Alert";
+import { useFragment, graphql } from "react-relay";
+import { MappedFuelTypeTable_normalizedFuelType$key } from "__generated__/MappedFuelTypeTable_normalizedFuelType.graphql";
+import { useDeleteFuelMappingMutation } from "mutations/fuelManagement/deleteFuelMapping";
+
+interface Props {
+  normalizedFuelType: MappedFuelTypeTable_normalizedFuelType$key;
+}
+
+export const MappedFuelTypeTable: React.FC<Props> = ({
+  normalizedFuelType,
+}) => {
+  const { fuelMappingsByFuelCarbonTaxDetailId } = useFragment(
+    graphql`
+      fragment MappedFuelTypeTable_normalizedFuelType on FuelCarbonTaxDetail {
+        fuelMappingsByFuelCarbonTaxDetailId(first: 2147483647)
+          @connection(
+            key: "MappedFuelTypes_fuelMappingsByFuelCarbonTaxDetailId"
+          ) {
+          __id
+          edges {
+            node {
+              id
+              fuelType
+            }
+          }
+        }
+      }
+    `,
+    normalizedFuelType
+  );
+
+  const [deleteFuelMapping] = useDeleteFuelMappingMutation();
+
+  const handleRemove = (id: string) => {
```

Comment:
After deleting a fuel mapping, I was expecting to see it in the "Unmapped Fuels" list, but didn't

Reply:
Deleting a fuel mapping will show up in the "Unmapped Fuels" list only if the fuel exists in the fuel table. There will be no data in swrs.fuel locally unless you add it.